src/sdgio-edit.owl

```diff
@@ -1608,6 +1645,291 @@ AnnotationAssertion(<http://www.geneontology.org/formats/oboInOwl#hasExactSynony
 AnnotationAssertion(rdfs:label <http://purl.unep.org/sdg/SDGIO_00020300> "CGIAR cross-cutting objective"@en)
 SubClassOf(<http://purl.unep.org/sdg/SDGIO_00020300> <http://purl.unep.org/sdg/SDGIO_00000251>)
 
+# Class: <http://purl.unep.org/sdg/SDGIO_00020308> (planned process to reduce poverty)
```

Comment:
This is a very hard definition. I had discussions about that but got nothing really useful. My main worry is that now the definition seems circular (roughly: reduction of poverty is ... rising population above the poverty line).

Reply:
Just to clarify, poverty, in some UN documents, is considered not just dependent on material resources (especially money) available now. It also depends on vulnerability (how easy is it for one to lose essential resource), education, etc. Your idea of using "basic goods" may allow us to find a solution though. What about:

"A planned process during which an agent - through either economic, technological, political, or societal means - secures stable access to basic goods and services, employment, or other resources or opportunities by which a human population can satisfy the bare necessities of life."

Reply:
> the definition seems circular (roughly: reduction of poverty is ... rising population above the poverty line)
 
 There is an operational definition of the poverty line that is used by UN Stats, but I see your point. 

>"A planned process during which an agent - through either economic, technological, political, or societal means - secures stable access to basic goods and services, employment, or other resources or opportunities by which a human population can satisfy the bare necessities of life."

Satisfying the bare necessities of life is not always enough to escape an impoverished state. 


https://en.wikipedia.org/wiki/Poverty#Definitions_and_etymology
> United Nations: Fundamentally, poverty is the inability of having choices and opportunities, a violation of human dignity. It means lack of basic capacity to participate effectively in society. It means not having enough to feed and clothe a family, not having a school or clinic to go to, not having the land on which to grow one's food or a job to earn one's living, not having access to credit. It means insecurity, powerlessness and exclusion of individuals, households and communities. It means susceptibility to violence, and it often implies living in marginal or fragile environments, without access to clean water or sanitation.[26]
>
>World Bank: Poverty is pronounced deprivation in well-being, and comprises many dimensions. It includes low incomes and the inability to acquire the basic goods and services necessary for survival with dignity. Poverty also encompasses low levels of health and education, poor access to clean water and sanitation, inadequate physical security, lack of voice, and insufficient capacity and opportunity to better one's life.[27]

Could you follow up that Wiki article and find the latest UN / World Bank definitions?

The definition would then go in the direction of  "...by which a human population can maintain and improve their health, well-being, security, dignity, education, and ability to participate in society."

Reply:
New iteration: A planned process during which an agent - through either economic, technological, political, or societal measures - causes or contributes to people acquiring and maintaining the capacity to appropriately take care of their health, well-being, security, dignity, education, and ability to participate in society.

Note: I say "acquiring and maintaining the capacity to appropriately take care etc." because there is no guarantee even for a rich that she will acquire or maintain her health, well-being, security, etc. 

Is that ok?